src/desktop/components/artist_page_cta/view.coffee

```diff
@@ -162,13 +160,17 @@ module.exports = class ArtistPageCTAView extends Backbone.View
     Mailcheck.run('#js-mailcheck-input-modal', '#js-mailcheck-hint-modal', false)
 
   trackImpression:(triggerType, contextModule) =>
-    analytics.track("Auth Impression", {
+    persistData = {
       modal_copy: "Join Artsy to discover new works by #{@artist.get('name')} and more artists you love",
-      onboarding: true,
       trigger: if triggerType == "scroll" then "scroll" else triggerType,
-      triggerSeconds: 4 if triggerType is "scroll",
+      trigger_seconds: 4 if triggerType == "scroll",
       type: "signup",
       intent: "signup",
       context_module: contextModule,
       auth_redirect: location.href
-    })
+    }
+
+    Cookies.set('artist-page-account-creation-data', JSON.stringify(persistData), { expires: 600 })
```

Comment:
I'm setting a cookie here because when we do the account creation (which includes some subsequent request) we lose access to some of the analytics data that is needed for the "created account" event.

Reply:
This is all being handled client-side tho, right? In that case, could the data just be saved in an variable here? Like `@accountData = ...`, and then later on `analytics.trigger 'signup', @accountData...` kinda thing.

Reply:
Yeah I tried making eventType an instance variable but I think that wasn't actually preserving the event type. I went ahead and tried this and it seem to work as I'd like.

Going to push that commit.